packages/manager/apps/pci-rancher/src/utils/rancherPrices.test.ts

```diff
@@ -39,14 +40,14 @@ describe('Rancher Prices Utilities', () => {
     });
   });
 
-  describe('calculateMonthlyPrice', () => {
+  describe('convertHourlyPriceToMonthly', () => {
     it('should calculate monthly price from hourly price', () => {
-      const result = calculateMonthlyPrice(100);
-      expect(result).toBe(72000);
+      const result = convertHourlyPriceToMonthly(100);
+      expect(result).toBe(EXPECTED_MONTHLY_PRICING);
     });
 
     it('should return 0 if hourly price is 0', () => {
-      const result = calculateMonthlyPrice(0);
+      const result = convertHourlyPriceToMonthly(0);
       expect(result).toBe(0);
     });
   });
```

Comment:
I'm not sure this is needed as the `convertHourlyPriceToMonthly` is already under unit test through the lib.